scripts/DeployNode.sh

```diff
@@ -1,26 +1,35 @@
+#!/usr/bin/env bash
+
+# Usage:
+# ./DeployNode.sh [GCP machine type, defaults to n1-highmem-8]
+#
+# Example:
+# ./DeployNode.sh n1-standard-4
+
 # This script has the following preconditions:
-# 1. You have a GCE SSD disk called node-disk with enough space for a parity db.
+# 1. You have a GCE SSD disk called node-disk with enough space for a geth db.
 # 2. node-disk is writable by most/all users (rw filesystem permissions have been granted).
-# 3. node-disk contains a folder called io.parity.ethereum (can be empty).
+# 3. node-disk contains a folder called ethereum (can be empty).
+
+MACHINE_TYPE=$1
+: ${MACHINE_TYPE:=n1-highmem-8}
 
-gcloud compute instances create-with-container custom-node \
-    --container-image docker.io/openethereum/openethereum:latest \
+gcloud compute instances create-with-container geth-node \
+    --container-image docker.io/ethereum/client-go \
     --zone northamerica-northeast1-b \
     --container-restart-policy on-failure \
     --container-stdin \
     --scopes cloud-platform \
     --disk=auto-delete=no,name=node-disk \
     --container-mount-disk=mount-path=/node-disk \
-    --machine-type n1-standard-2 \
+    --machine-type $MACHINE_TYPE \
     --container-privileged \
-    --container-arg="--chain" \
-    --container-arg="mainnet" \
-    --container-arg="--ipc-path" \
-    --container-arg="./ipc" \
-    --container-arg="--warp-barrier" \
-    --container-arg="9700000" \
-    --container-arg="--no-ancient-blocks" \
-    --container-arg="-d" \
-    --container-arg="/node-disk/io.parity.ethereum" \
-    --container-arg="--ws-interface=all" \
-    --container-arg="--jsonrpc-interface=all"
+    --container-arg="--rpc" \
+    --container-arg="--rpcaddr" \
+    --container-arg="0.0.0.0" \
+    --container-arg="--ws" \
+    --container-arg="--wsaddr" \
+    --container-arg="0.0.0.0" \
+    --container-arg="--ipcdisable" \
+    --container-arg="--datadir" \
+    --container-arg="/node-disk/ethereum"
```

Comment:
do we not want to add the fast sync mode `--syncmode=fast`?

Reply:
I think fast is the default: https://geth.ethereum.org/docs/interface/command-line-options?

Reply:
ok great